external/js/admin.js

```diff
@@ -22,7 +22,7 @@ $(document).ready(function(){
 	}
 
 	function saveSites() {
-		var post = $('#external').serialize();
+		var post = $('#external form').serialize();
```

Comment:
You're right. Actually the `#external` div is generated by the admin page itself, where "external" is the name of the app.

So I'll rather remove the id on the `form`